docs/source/terms_and_model.rst

```diff
@@ -669,8 +669,9 @@ both current and future.  Variation subclasses are:
 
 **Biological definition**
 
-In biology, variation is often used to mean `genetic variation`_,
-describing the differences observed in DNA among individuals.
+In biology, variation is often used to mean *sequence* variation,
```

Comment:
The lines above this change has. "Variation subclasses are:" and then has "Biological definition"

Reply:
Fixed.